gstcefsrc.cc

```diff
@@ -778,13 +789,22 @@ gst_cef_src_start(GstBaseSrc *base_src)
   GST_OBJECT_UNLOCK (src);
```

Comment:
The `g_cond_wait()` 10 lines above would also wait forever if we end up here on macOS on the main thread or not?

Reply:
Not after this MR. If GStreamer is managed from the main thread, this function gets called by line 762 above, by which time either CEF has initialized (skipping the wait altogether), or it has failed (and this function will not be called at all). The wait is still necessary if the app handles CEF in a different event loop from the GStreamer pipeline's, like with `gst-launch-1.0`.